python/cudf/cudf/fuzz_tests/fuzz_test_parquet.py

```diff
@@ -19,6 +20,34 @@ def parquet_reader_test(parquet_buffer):
     assert_eq(gdf, pdf)
 
 
+@pythonfuzz(
+    data_handle=ParquetReader,
+    params={
+        "columns": None,
+        "engine": ["pyarrow", None],
+        "use_pandas_metadata": [True, False],
+    },
+)
+def parquet_reader_columns_engine(
+    parquet_buffer, columns, engine, use_pandas_metadata
+):
+    print("PRAMS", columns, engine, use_pandas_metadata)
+    pdf = pd.read_parquet(
+        parquet_buffer,
+        columns=columns,
+        engine=engine,
+        use_pandas_metadata=use_pandas_metadata,
+    )
+    gdf = cudf.read_parquet(
+        parquet_buffer,
+        columns=columns,
+        engine=engine,
+        use_pandas_metadata=use_pandas_metadata,
+    )
+
+    assert_eq(gdf, pdf)
```

Comment:
Given Pandas will use the PyArrow parquet reader and then convert nullable integer / boolean columns to floating point columns, it allows for bugs to silently slip by here. Would it be possible to just use the PyArrow parquet reader directly and then check for equality between the PyArrow Table and the cuDF DataFrame?

Reply:
Resolving this as I had a discussion offline with @kkraus14 regarding this and pandas seems to be preserving nullable types:

```python
In[2]: import pandas as pd
In[3]: s = pd.Series([1, 2, 3, None, 3], dtype=pd.Int64Dtype())
In[4]: df = pd.DataFrame({'a':s})
In[5]: df
Out[5]: 
      a
0     1
1     2
2     3
3  <NA>
4     3
In[6]: df.to_parquet('a')
In[7]: pd.read_parquet('a')
Out[7]: 
      a
0     1
1     2
2     3
3  <NA>
4     3
In[8]: pd.read_parquet('a').dtypes
Out[8]: 
a    Int64
dtype: object
In[9]: pd.read_parquet('a', engine='pyarrow')
Out[9]: 
      a
0     1
1     2
2     3
3  <NA>
4     3
In[10]: pd.read_parquet('a', engine='pyarrow').dtypes
Out[10]: 
a    Int64
dtype: object
```